src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java

```diff
@@ -362,6 +385,7 @@ private void handlePartiallyFailedElasticInsert(
           }
         }
     );
+    rollbackHandleCreation(rollbackDigitalRecords);
```

Comment:
Let's see how this works, as there was a kind of flow in the other ones -> rollback elastic -> rollback db -> rollback handle -> DLQ. Now it is rollback elastic -> rollback db -> DLQ -> rollback handle. Not sure if that could impact anything.

Reply:
Handles are done at the end because we DLQ objects one at a time, but handle rollbacks should be done as one batch.